Sming/compiler/ld/eagle.app.v6.cpp.include.ld

```diff
@@ -0,0 +1,217 @@
+/*  
```

Comment:
> Well, the tests seem to fail because they do not see the new file eagle.app.v6.cpp.include.ld

Not really. You should add the compiler/ld directory also for normal projects. A sample patch is given below.

```patch
diff --git a/Sming/Makefile-project.mk b/Sming/Makefile-project.mk
index 4e37697..503471a 100644
--- a/Sming/Makefile-project.mk
+++ b/Sming/Makefile-project.mk
@@ -365,7 +365,7 @@ spiff_update: spiff_clean $(SPIFF_BIN_OUT)
 
 $(TARGET_OUT): $(APP_AR)
        $(vecho) "LD $@"        
-       $(Q) $(LD) -L$(USER_LIBDIR) -L$(SDK_LIBDIR) $(LD_SCRIPT) $(LDFLAGS) -Wl,--start-group $(LIBS) $(APP_AR) -Wl,--end-group -o $@
+       $(Q) $(LD) -L$(USER_LIBDIR) -L$(SDK_LIBDIR) -L$(SMING_HOME)/compiler/ld $(LD_SCRIPT) $(LDFLAGS) -Wl,--start-group $(LIBS) $(APP_AR) -Wl,--end-group -o $@
 
        $(Q) $(STRIP) $@
 ```